intro-to-R-tidyverse/03-intro_to_tidyverse.Rmd

```diff
@@ -262,13 +275,14 @@ Let's take a look at what using `%in%` does.
 
 ```{r in-example, eval = FALSE}
 genes_of_interest <- c("SNCA", "CDKN1A")
-stats_df$gene_symbol %in% genes_of_interest
+# Are these genes present in the `gene_symbol` column in stats_df?
+genes_of_interest %in% stats_df$gene_symbol
```

Comment:
The reason this was the other way around was that this was that  question we were interested in. Which rows pertain our gene of interest, not whether our genes of interest are in the data set. The fact that it gives a very long vector is perhaps unfortunate (and why this chunk is not evaluated by default), but the idea was to show what the filter statement is actually seeing.

I think showing   `genes_of_interest %in% stats_df$gene_symbol` first probably makes sense (but maybe we want to include a gene that is _not_ present too?), but I think we may want some additional scaffolding to discuss why `filter()` uses the other direction.

Reply:
I'm going to come back to this PR shortly. Will revert this particular change and then re-request review.